sharding-proxy/sharding-proxy-backend/src/main/java/org/apache/shardingsphere/shardingproxy/backend/text/sctl/hint/internal/executor/HintShowTableStatusExecutor.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.shardingproxy.backend.text.sctl.hint.internal.executor;
+
+import com.google.common.base.Joiner;
+import lombok.RequiredArgsConstructor;
+import org.apache.shardingsphere.api.hint.HintManager;
+import org.apache.shardingsphere.core.merge.MergedResult;
+import org.apache.shardingsphere.core.merge.dal.show.ShowShardingCTLMergedResult;
+import org.apache.shardingsphere.core.rule.TableRule;
+import org.apache.shardingsphere.shardingproxy.backend.communication.jdbc.connection.BackendConnection;
+import org.apache.shardingsphere.shardingproxy.backend.response.query.QueryHeader;
+import org.apache.shardingsphere.shardingproxy.backend.text.sctl.hint.internal.result.HintShowTableStatusResult;
+
+import java.sql.Types;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Hint show table status command executor.
+ *
+ * @author liya
+ */
+@RequiredArgsConstructor
+public final class HintShowTableStatusExecutor extends AbstractHintQueryExecutor {
+    
+    private final BackendConnection backendConnection;
+    
+    protected List<QueryHeader> createQueryHeaders() {
+        List<QueryHeader> queryHeaders = new ArrayList<>(3);
+        queryHeaders.add(new QueryHeader("", "", "table_name", "", 255, Types.CHAR, 0));
+        queryHeaders.add(new QueryHeader("", "", "database_sharding_values", "", 255, Types.CHAR, 0));
+        queryHeaders.add(new QueryHeader("", "", "table_sharding_values", "", 255, Types.CHAR, 0));
+        return queryHeaders;
+    }
+    
+    protected MergedResult createMergedResult() {
```

Comment:
we can simplify this method, image like this:
  1. getAllLogicTables from shardingRule
  2. iterate all logicTables to generate results
      it seems that 3 fill method can be merged to one